src/mainwindow.cpp

```diff
@@ -7294,6 +7294,7 @@ void MainWindow::on_noteFolderComboBox_currentIndexChanged(int index) {
 
     if (noteFolder.isFetched()) {
         changeNoteFolder(noteFolderId);
+        areBrokenTagNoteLinksRemoved = false;
```

Comment:
you can also switch the note folder via the settings dialog and the `Note / Note folders` menu (you can hide the note folder selector panel), is this line also triggered in those cases?

Reply:
And can you please start all new global variables with an underline, `_areBrokenTagNoteLinksRemoved` in this case. :grin:

Reply:
Done